app/views/search/advanced/index.html.erb

```diff
@@ -1 +1,48 @@
-<h1>Advanced Search Options</h1>
+<div class="container">
+  <div class="col-sm-9 col-md-7 col-lg-6 mx-auto">
+    <br><h1 class="card-title text-center">Advanced Search Options</h1><br>
```

Comment:
The advanced search options form is made, but not 100% styled. TBD if this will actually work for how we are building out search, but I'm pretty sure these are the fields we will be able to search by.